lib_comfyui/comfyui_adapter.py

```diff
@@ -19,18 +19,31 @@ def unwrap_cpu_state_dict(state_dict: dict) -> dict:
     }
 
 
+def get_opts_outdirs():
+    return {
+        'outdir_samples': shared.opts.outdir_samples,
+        'outdir_grids': shared.opts.outdir_grids,
+        'outdir_txt2img_samples': shared.opts.outdir_txt2img_samples,
+        'outdir_img2img_samples': shared.opts.outdir_img2img_samples,
+        'outdir_extras_samples': shared.opts.outdir_extras_samples,
+        'outdir_txt2img_grids': shared.opts.outdir_txt2img_grids,
+        'outdir_img2img_grids': shared.opts.outdir_img2img_grids,

```

Comment:
is there a reason not to simply return `shared.opts`?

Reply:
yes, turns out there are functions in it, and pickle doesn't want to do it

Reply:
Should I try to prune it to make pickle happy?

Reply:
I thought the options were serializable. There is probably a function in the webui that turns them into json, we should be able to rely on this feature to extract just the subset that is serializable.

Reply:
You're right, there's a way to do it. Thanks for pointing that out